### DISCUSSION TITLE
利用規約のMarkdown化

### DISCUSSION THREAD

**Discussion by haruharu-1105** [aipictors/aipictors#63] [アイデア - Ideas]
## どのような問題がありますか？
現状の[利用規約](https://www.aipictors.com/terms/)は課題として

- 利用規約以外のコードが出力されている。
view-source:https://www.aipictors.com/terms/

- 規約の版管理が行えていない。

- 利用規約の文内にリンクが貼れない。
→規約参照時のポインタとして不便です。

## どうすれば良いですか？
Github内にMarkdownで管理しつつ、そこから生成するのはどうでしょうか？

**Comment by shigurenimo** [comment]
それが良いと思う。

**Comment by shigurenimo** [comment]
アプリに表示される利用規約のページはマークダウンになってる。 

https://github.com/aipictors/aipictors/blob/main/app/%5Blang%5D/flutter/terms/page.tsx

https://beta.aipictors.com/flutter/terms
